src/pages/TravelRestrictions/index.js

```diff
@@ -0,0 +1,38 @@
+import i18next from "i18next";
+import _ from "lodash";
+
+const travelRestrictionsHelper = (elementId, countries) => {
+  const countryList = [];
+  // Iterate through and render country links
+  _.orderBy(countries, "name", "desc").map((country) => {
```

Comment:
If using only `orderBy` in this file it'll be better to have 
```
import { orderBy }  from "lodash";
```
instead of importing the whole lodash